README.md

```diff
@@ -1 +1,70 @@
-# math-magician
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly update your README with [Microverse template](https://github.com/microverseinc/readme-template) and update it with your info instead of react template in order to make your README more professional.